src/api/index.js

```diff
@@ -360,6 +361,17 @@ export function fetchCommitHistory(payload) {
   return ajax.get(url, { ...payload });
 }
 
+export function deleteBot(payload) {
+  const { name, language, group } = payload;
```

Comment:
Same here

Reply:
@fragm3 in the deleteBot() function in BotBuilder.js the argument received is as name, so I am passing it as name in the apis.deleteBot function.

Reply:
Use name: skill

Reply:
@fragm3 Cool will make the changes.